cli/silence_import.go

```diff
@@ -123,7 +119,7 @@ func (c *silenceImportCmd) bulkImport(ctx context.Context, _ *kingpin.ParseConte
 
 	count := 0
 	for dec.More() {
-		var s types.Silence
+		var s models.PostableSilence
```

Comment:
It needs to be mentioned somewhere that silence backups created prior to this PR won't be supported after.

Reply:
I did a bit more testing around this and basic backwards compatibility seems to work ok.  I exported silences using the v1 amtool, and then imported the file using the v2 amtool.  I'll also add a note to the changelog about possible incompatibility.